agent/engine/docker_task_engine_windows_test.go

```diff
@@ -504,34 +504,32 @@ func TestPauseContainerHappyPath(t *testing.T) {
 	dockerClient.EXPECT().CreateContainer(gomock.Any(), gomock.Any(), gomock.Any(),
 		gomock.Any(), gomock.Any()).Return(dockerapi.DockerContainerMetadata{DockerID: sleepContainerID2})
 
-	gomock.InOrder(
```

Comment:
Any reason to remove the `InOrder` statement?

Reply:
These API calls are called in parallel and it's not necessary that the order is followed.
In the Linux counterpart of this test, we do not have any ordering.